default/views.py

```diff
@@ -18,31 +18,41 @@
 from requests.exceptions import ConnectionError, HTTPError, SSLError
 
 from default.data_file import DataFile
-from default.decorators import clear_files, published_date, require_files
+from default.decorators import clear_drive_session_vars, clear_files, published_date, require_files
 from default.forms import MappingSheetOptionsForm
+from default.google_drive import get_credentials_from_session, google_api_messages, upload_to_drive
 from default.mapping_sheet import (get_extended_mapping_sheet, get_mapping_sheet_from_uploaded_file,
                                    get_mapping_sheet_from_url)
 from default.util import (get_files_from_session, invalid_request_file_message, json_response, make_package,
                           ocds_command)
 
 
-def retrieve_result(request, folder, id, format=None):
+def get_datafile_filename(folder, id, format):
     if format is None:
         prefix = 'result'
         ext = '.zip'
         filename = 'result.zip'
     elif format == 'csv':
+        prefix = 'result'
+        ext = '.csv'
+        filename = 'result.csv'
+    elif format == 'csv.zip':
         prefix = 'flatten-csv'
         ext = '.zip'
-        filename = 'result-csv.zip'
+        filename = 'result.csv.zip'
```

Comment:
We must update the file name 'result-csv.zip' in the message box.

![image](https://user-images.githubusercontent.com/48607824/87622315-dd91ec00-c6f0-11ea-8c69-39d6fc4016dd.png)